code/obj.dm

```diff
@@ -14,6 +14,9 @@
 	var/move_triggered = 0
 	var/object_flags = 0
 
+	/// for if a mimic has trapped the obj, for things like crates and chutes
+	var/mob/living/critter/mimic/antag_spawn/present_mimic = null
```

Comment:
Don't put this on `/obj`, make it a component like `/datum/component/hidden_mimic` orr wrap it into the current mimic stomach component and add that to the storage object instead of the mimic (that probably makes more sense anyway?)